tests/logictest/suites/base/09_fuse_engine/09_0001_remote_insert_v2

```diff
@@ -47,9 +47,6 @@ SELECT * FROM t2;
 1 2021-09-07 21:38:35.000000 2021-09-07
 0 2021-09-07 21:38:35.000000 2021-09-07
 
- 2021-09-07 21:38:35.000000 2021-09-07
```

Comment:
ditto

Reply:
This will use the default value? ping @dantengsky

Reply:
those are more like typos to me.  seems the commit forget to remove the deprecated “expected resultset” after tuning the test case.

do not know how will the “logictest driver” interpret them..